mesheryctl/cmd/vars.go

```diff
@@ -131,3 +131,33 @@ func installprereq() {
 	}
 	log.Info("Prerequisite Docker Compose is installed.")
 }
+
+func isMesheryRunning() bool {
+	ps := exec.Command("docker-compose", "-f", dockerComposeFile, "ps")
+	grep := exec.Command("grep", "meshery")
```

Comment:
@mayankshah1607, this is an improvement and works on macOS and Linux, but is an area where we will either need to leverage a universally available tool (e.g. parse the output in go) or will need to check for OS arch and ensure that an OS-specific utility is used for Windows, macOS and Linux compatibility of `mesheryctl`.

Reply:
@leecalcote yep, that's a good one. Should have thought of that. Will get back shortly with a fix :)